rpcs3/Emu/Cell/PPUThread.h

```diff
@@ -163,6 +164,9 @@ class ppu_thread : public cpu_thread
 
 	using cpu_thread::operator=;
 
+	// Hypervisor context data
+	alignas(16) rpcs3::hypervisor_context_t hv_ctx; // HV context for gate enter exit. Keep at a low struct offset.
```

Comment:
Oh, this would require invalidating existing PPU cache IIRC

Reply:
Indeed. I forgot about that.

Reply:
Maybe #ifdef it so x86 has hv_ctx at previous RSP-saving position

Reply:
Done. Previous cache loads fine from testing.